packages/services-store/src/types/translateAPI.ts

```diff
@@ -85,13 +85,52 @@ export interface MastheadLink {
 
 /**
  * A menu section for masthead
+ * @deprecated
  */
-export interface MastheadL1 {
+export interface LegacyMastheadL1 {
   title: string;
   url?: string;
   menuItems?: MastheadLink[];
 }
 
+/**
+ * A menu section for masthead
+ */
+export interface MastheadL1 {
+  title: string;
+  url?: string;
+  menuItems: L1MenuItem[];
+  actions?: {
+    cta?: BasicLink;
+    login?: BasicLink;
+  };
+}
+
+export interface L1MenuItem extends BasicLink {
+  submenu?: L1Submenu;
+}
+
+export interface L1Submenu {
+  announcement?: string; // From AEM rich text editor
```

Comment:
Now that we're working with actual code, can we make this something like `HTMLTemplateResult | HTMLElement | Text`. I considered simply `Node`, but I feel like that's too generic. What do you think?

Reply:
I don't know that we want to define the JSON data as HTMLTemplateResult or HTMLElement. I think that value will always be a `string`, even if that string contains markup.

Reply:
You're right. I was thinking about how these types are enforced in the component code, but that wouldn't work with the JSON that is our actual content source.